clean_up_dynamo/handler.py

```diff
@@ -213,6 +213,30 @@ def find_item_records_with_images(table_name: str):
     return unique_imageGroupIds_referenced
 
 
+def delete_certain_image_records(table_name: str):
+    """ Delete certain File records, depending on criteria below """
+    print("deleting Image records")
+    pk = 'IMAGE'
+    sk = 'IMAGE#DIGITAL/MARBLE-IMAGES'
+    kwargs = {}
+    kwargs['KeyConditionExpression'] = Key('PK').eq(pk) & Key('SK').begins_with(sk)
```

Comment:
```suggestion
    kwargs['KeyConditionExpression'] = Key('PK').eq(pk) and Key('SK').begins_with(sk)
```

Reply:
Thanks for the suggestion.  However, the example I followed from AWS used "&" instead of "and", so I replicated that.  I have used the "&" format on all the queries in this module, and they all work.  

I am only including this clean_up_dynamo module in source control for future reference, in case we need to write scripts to do more things directly in dynamo.  This code is never deployed, and is only intended to be run interactively.